Makefile

```diff
@@ -32,11 +34,14 @@ ifeq ($(UNAME_S),Darwin)
 endif
 TMPFILE := $(shell mktemp)
 
+GOOSALT ?= 'linux'
+GOOS='$(GOOSALT)'
+
 LOCALIZATION_FILES := $(shell find . -name \*.toml | grep -v vendor | grep -v ./bin)
 
 $(NAME): $(shell find . -name \*.go)
-	CGO_ENABLED=0 go build $(TAGS) -ldflags "$(VFLAG)" -o build/$@ .
-
+	CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=amd64 go build $(TAGS) -ldflags "$(VFLAG)" -o build/$@ .
```

Comment:
I know you are copying from what is in quake-client, however I'm not sure that GOARCH=amd64 will always work. I think others with Macs need GOARCH=aarch64. Let me ask.

Reply:
Yes, we will need to support more than amd64 . For now we can go with this one and then I will create an issue to support additional GOARCH input.

Reply:
Ok, thank you